format/nexus.py

```diff
@@ -1085,74 +1110,49 @@ def find_scanning_axis(obj):
             return o
 
 
-class ScanFactory(object):
```

Comment:
This pseudoclass was turned into a function. Fully support this, however in the context of a stable release this is an API change on a public name. I will put the class back in in a separate commit.

Reply:
done